Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:stable-alpine
```

Comment:
- PIN via sha256 hash to specific version
  - Add comment above showing that the tag was mapped to stable-alpine 
- Add link to the tag in the docker registry, from which you pull the image

The reason you want to do that is that builds should produce the same image all the time. i.e. reproducable builds.

In your case image you will build now in 2 months later will be different, which is bad for maintenance and trobleshooting.

Reply:
Done